designs/virtualization/4.5/Scheduling/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,25 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Firmware Config
+
+Exposing Firmware Config for VM.
+
+![VM page](img/Firmware-0-0.jpg)
+Firmware Config is placed in the Scheduling sction of the VM details tab.
+The default value of this field is "Default"
+
+![Config modal](img/Firmware-1-0.jpg)
+This modal is composed by three fields:
+- Bootload (Selection): BIOS (Default) and EFI
+- Serial (sting): [The system-serial-number in SMBIOS]
+- UUID ( sting): [UUID reported by the vmi bios. Defaults to a random generated UID]
+
+![Config modal](img/Firmware-1-1.jpg)
```

Comment:
should we offer field level help on these?

Reply:
I'll work on it

Reply:
I added and committed the info appearing on this page with a link to it (see in the MD).
https://kubevirt.io/api-reference/v0.26.0/definitions.html#_v1_firmware
Waiting for an answer to hear if it is sufficient enough form Gilad

Reply:
@matthewcarleton , @glekner said he's ok with the text